forge-gui/res/cardsfolder/upcoming/souls_of_the_lost.txt

```diff
@@ -0,0 +1,10 @@
+Name:Souls of the Lost
+ManaCost:1 B
+Types:Creature Spirit
+PT:*/1+*
```

Comment:
*+1 not 1+*

Reply:
Thanks for clarifying. I think I used Tarmogoyf's declaration, which is written as `1+*`. I'll keep this in mind